src/main/java/org/cyclonedx/maven/BaseCycloneDxMojo.java

```diff
@@ -119,6 +120,9 @@
     @Parameter(property = "outputFormat", defaultValue = "all", required = false)
     private String outputFormat;
 
+    @Parameter(property = "outputDirectory", defaultValue = "target", required = false)
```

Comment:
as a Maven plugin writer, when we add such a property, we usually define `${project.build.directory}` as default value for target:
- this avoid copy/pasting `target`
- this permits the user to eventually save to an external directory (see next comment)
see for example https://maven.apache.org/plugins/maven-dependency-plugin/copy-dependencies-mojo.html#outputDirectory 

and adding a `@since` tag helps users discover when this parameter was added

Reply:
After looking at this some more I believe the suggestion would make this plugin inconsistent with the other plugins which inspired me to make this change in the first place.  I'm referring to the ossindex-maven-plugin, the maven-dependency-plugin and the dependency-check-maven plugin which all take output paths relative to the project folder.  Thoughts?

Reply:
Agreed. Adding `@since` would be good and defining the default as  `${project.build.directory}` should also be done.